.claude/skills/diagnose-maestro-deployment/scripts/diagnose.sh

```
@@ -0,0 +1,564 @@
+
+# Function to extract cluster info from deployment output
+extract_cluster_info() {
+    local output_file=$1
+
+    echo "Analyzing deployment output: $output_file"
+    echo ""
+
+    # Try to find cluster names from the output
+    if grep -q "pers-usw3" "$output_file"; then
+        # Extract cluster name pattern
+        local cluster_base=$(grep -o "pers-usw3[a-z]*" "$output_file" | head -1)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)